src/utils/validators.js

```diff
@@ -7,7 +7,8 @@ const SOCIAL_MEDIA_REGEX_PART_2 = '.com/)([a-zA-Z0-9_-]+(/)?)+$';
 const permalinkRegexTest = RegExp(PERMALINK_REGEX);
 const DATE_REGEX = '^([0-9]{4}-[0-9]{2}-[0-9]{2})$';
 const dateRegexTest = RegExp(DATE_REGEX);
-const fileNameRegexTest = /^[^<>:;,?"*|/]+\.[a-zA-z]{3,4}$/;
+const fileNameRegexTest = /^[^<>:;,?"*|/]+$/;
```

Comment:
This regex currently still allows a lot of special characters, we might want to change it to only allow characters, numbers, _ and -

Reply:
resolved with [3b8ef00](https://github.com/isomerpages/isomercms-frontend/pull/168/commits/3b8ef002a5e71b0bea9aec14c67bbe87ff2f8835)

Reply:
Actually this still allows ".", since everything afterwards is matched to .+

We could possibly do the extension check first, then split on the "."?

Reply:
fixed with [84afce0](https://github.com/isomerpages/isomercms-frontend/pull/168/commits/84afce0e23e554d532d60f7c268bf2a6fb2f4f4c)